test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -637,3 +632,18 @@ func ensureDesiredReplicasInRange(ctx context.Context, deploymentName, namespace
 	}
 	framework.ExpectNoErrorWithOffset(1, err)
 }
+
+func noExporterDeployment(name, namespace string, replicas int32) *appsv1.Deployment {
+	d := e2edeployment.NewDeployment(name, replicas, map[string]string{"name": name}, "", "", appsv1.RollingUpdateDeploymentStrategyType)
+	d.ObjectMeta.Namespace = namespace
+	d.Spec.Template.Spec = v1.PodSpec{Containers: []v1.Container{
+		{
+			Name:            "sleeper",
+			Image:           "k8s.gcr.io/ubuntu-slim:0.1",
```

Comment:
you have the agnhost image that is multiplatform and already has a pause command https://pkg.go.dev/k8s.io/kubernetes/test/images/agnhost#readme-pause

Reply:
thanks!